lesson_1

katelarionova2058 opened the pull request.

katelarionova2058 committed:
lesson_1

VidgarVii commented on config/initializers/version.rb:
```diff
@@ -0,0 +1,3 @@
+module TestGuru
+  VERSION = '0.1.0'
```
"мутабельные" объекты (строки, хэши, массивы) рекомендуется `freeze`-ить, чтобы они были неизменяемыми
в противном случае константу, ссылающуюся на "мутабельный" объект,  можно изменить: `VERSION << '3.14..'`

katelarionova2058 committed:
new lesson

katelarionova2058 committed:
новый файл:    app/models/answer.rb
	новый файл:    app/models/category.rb
	новый файл:    app/models/guestion.rb
	новый файл:    app/models/test.rb
	новый файл:    app/models/user.rb
	новый файл:    db/migrate/20220215220058_create_tests.rb
	новый файл:    db/migrate/20220215235112_create_users.rb
	новый файл:    db/migrate/20220217203228_create_guestions.rb
	новый файл:    db/migrate/20220217203404_create_answers.rb
	новый файл:    db/migrate/20220217203640_create_categories.rb
        новый файл:    db/migrate/20220217221102_add_default_value.rb
	новый файл:    db/schema.rb
	новый файл:    test/fixtures/answers.yml
	новый файл:    test/fixtures/categories.yml
	новый файл:    test/fixtures/guestions.yml
	новый файл:    test/fixtures/tests.yml
	новый файл:    test/fixtures/users.yml
	новый файл:    test/models/answer_test.rb
	новый файл:    test/models/category_test.rb
	новый файл:    test/models/guestion_test.rb
	новый файл:    test/models/test_test.rb
	новый файл:    test/models/user_test.rb

new lesson

katelarionova2058 committed:
Update version.rb

katelarionova2058 committed:
Update 20220215220058_create_tests.rb

katelarionova2058 committed:
Update 20220215235112_create_users.rb

katelarionova2058 committed:
Update 20220217203228_create_guestions.rb

katelarionova2058 committed:
Update 20220217203404_create_answers.rb

katelarionova2058 committed:
Update 20220217203640_create_categories.rb

katelarionova2058 committed:
Delete 20220217221102_add_default_value.rb

katelarionova2058 committed:
Update schema.rb

katelarionova2058 committed:
Update 20220215220058_create_tests.rb

katelarionova2058 committed:
Update schema.rb

katelarionova2058 committed:
Merge pull request #2 from katelarionova2058/lesson2

Lesson2
